src/sentry/static/sentry/app/views/settings/project/projectTeams.jsx

```diff
@@ -134,12 +166,40 @@ class ProjectTeams extends AsyncView {
   renderBody() {
     let body;
 
-    if (this.state.teams.length > 0) body = this.renderResults();
+    if (this.state.projectTeams.length > 0) body = this.renderResults();
     else body = this.renderEmpty();
 
+    let projectTeams = new Set(this.state.projectTeams.map(team => team.slug));
+    let teamsToAdd = this.state.allTeams.filter(team => {
+      return team.hasAccess && !projectTeams.has(team.slug);
+    });
+
     return (
       <div>
-        <SettingsPageHeader title={t('Teams')} />
+        <SettingsPageHeader
+          title={t('Teams')}
+          action={
+            <DropdownLink
+              anchorRight
+              className="btn btn-primary btn-sm"
+              title={t('Add Team')}
+            >
+              {teamsToAdd.length ? (
+                teamsToAdd.map(team => {
+                  return (
+                    <MenuItem noAnchor={true} key={team.slug}>
+                      <a onClick={this.handleAdd.bind(this, team)}>{team.name}</a>
```

Comment:
Is it easy to use `<Button priority="link"/>` here? if not don't worry about it

Reply:
it definitely messes with the styling a lot:
![screenshot 2018-01-31 15 32 19](https://user-images.githubusercontent.com/5026776/35653217-f6bbbba6-069b-11e8-91f2-f375fe9a5124.png)

Reply:
Damn, just leave it then